pkg/sql/sem/tree/create_logical_replication.go

```diff
@@ -168,12 +168,12 @@ func (o *LogicalReplicationOptions) CombineWith(other *LogicalReplicationOptions
 		}
 	}
 
-	if o.FilterRangefeed != nil {
-		if other.FilterRangefeed != nil {
+	if o.IgnoreCDCIgnoredTTLDeletes != nil {
+		if other.IgnoreCDCIgnoredTTLDeletes != nil {
 			return errors.New("IGNORE_TTL_DELETES option specified multiple times")
```

Comment:
Since you're already fixing this, can you also change the error to be IGNORE_CDC_IGNORED_TTL_DELETES

Reply:
Done.